modules/ocf/manifests/ssl/lets_encrypt/dns.pp

```diff
@@ -10,24 +10,30 @@
     content => "${join($domains, ' ')} > ${title}",
   }
 
-  # Only run the dehydrated command to renew the cert if it is old enough (30
-  # days = 2592000 seconds left until it expires). dehydrated does this check
-  # itself, but this should help with puppet error spam a bit (for instance
-  # if Let's Encrypt is down for any reason) and mean that services can
-  # subscribe to this class to reload after new certs are obtained since this
-  # won't refresh unless dehydrated is re-run for some reason.
-  exec { "check ${title} cert expiration":
-    command => '/bin/true',
-    unless  => "openssl x509 -checkend 2592000 -noout -in /var/lib/lets-encrypt/certs/${title}/cert.pem",
-    user    => ocfletsencrypt,
-  } ~>
+  # Only run the dehydrated command to renew the cert if it is old enough, or
```

Comment:
I think the comment is a little out of place now that it's separated from the `exec` it was talking about, although I see why you put it here since the checking logic is just below here. Maybe it should be split into two sections, one about checks to perform on the cert and wanting to re-obtain the cert if the cert is old or the domain has changed, and another section about the exec and the puppet spam? Something like that anyway, since some of this feels like it belongs further down.